binding.cc

```diff
@@ -479,6 +489,7 @@ struct Iterator {
   }
 
   ~Iterator () {
+    assert(database_->closing_); // NOTE: debugging
```

Comment:
To clarify https://github.com/Level/leveldown/pull/597#issuecomment-466811204: this should always be true, currently isn't.

Reply:
No. Nevermind. I need a break..